src/main/java/org/alliancegenome/curation_api/services/helpers/GenericOntologyLoadHelper.java

```diff
@@ -91,13 +91,15 @@ public T traverse(OWLClass parent, int depth) throws Exception {
 
             termParent = getOntologyTerm(parent);
 
-            String requiredNamespace = config.getAltNameSpace();
-            if (requiredNamespace == null) {
-                requiredNamespace = defaultNamespace;
+            ArrayList<String> requiredNamespaces = config.getAltNameSpaces();
+            if (requiredNamespaces.isEmpty()) {
+                if (defaultNamespace != null) {
+                    requiredNamespaces.add(defaultNamespace);
+                }
             }
             
             if(
-                (termParent.getNamespace() != null && requiredNamespace != null && termParent.getNamespace().equals(requiredNamespace)) ||
+                (termParent.getNamespace() != null && !requiredNamespaces.isEmpty() && requiredNamespaces.contains(termParent.getNamespace())) ||
```

Comment:
Also when using a list you wouldn't need to do the `!requiredNamespaces.isEmpty()` check as the contains on an empty list will work correctly.

Reply:
done